.github/workflows/build.yml

```diff
@@ -55,8 +62,45 @@ jobs:
           chmod +x ./scripts/setup_west_workspace.sh
           ./scripts/setup_west_workspace.sh
 
-      - name: Build app
+      - name: Builds
+        env:
+          GH_TOKEN: ${{ github.token }}
         run: |
+          set +e
```

Comment:
Can we do something like below and not apply `set +e` here?
```
set +e
west -v build -b ${{ matrix.board }} ${{ matrix.path }} -p
build_status=$?
set -e
```

We want to stop the workflow unless it's a build error.